content/60.methods.md

```diff
@@ -1,50 +1,101 @@
+Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
+The GitHub implementation allowed users to contribute either using git on the command line or using the GitHub user interface, and we developed documentation for users with less experience with this platform.
 
-#### Collaborative Writing and Manuscript Generation
+Few researchers in biological and medical fields are trained in version control tools such as git <!--To Do: Find some sort of reference for this, software carpentry?-->
+The current project was managed through GitHub [@url:https://github.com/greenelab/covid19-review] using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
+As a result, we expanded upon the baseline Manubot training materials in an effort to guide efforts by scientists with no background in GitHub or version control to contribute.
+<!--To Do: Short description of efforts to develop additional training resources-->
 
-Manubot [@doi:10.1371/journal.pcbi.1007128] is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
-Here, Manubot was used to generate a manuscript from text maintained using GitHub, a popular, online version control interface.
-The GitHub implementation allowed users to contribute either using git on the command line or using the GitHub user interface, and we developed documentation for users with less experience with this platform.
-Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
-Due to the needs of this project, project contributors also implemented new features in Manubot and [Zotero](https://www.zotero.org/), which Manubot uses to extract metadata for some types of citations.
-These features support directly citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899] and generating the complete review manuscript along with the individual manuscripts reviewing specific topics. <!-- TODO: reference individual manuscripts -->
-Finally, Manubot and GitHub Actions continuous integration allowed for scripted updates to be run each time the manuscript was generated.
-These scripts were used to check that the manuscript was built correctly, run spellchecking, and cross-reference the manuscripts cited in this review, summarized in Appendix A, and discussed in the project's issues and pull requests.
+The data regarding contributions was analyzed using GitHub's continuous integration features.
+<!--To Do: Describe how all the graphs we decide to go with were made, see results for ideas-->
 
-#### Data Analysis and Visualization
+#### Applying Manubot to COVID-19 
```

Comment:
We should emphasize that we were not just applying Manubot as-is.  The demands of this review and its scale required software, infrastructure, and workflow improvements.  For this type of conference, we should emphasize the methods development and contributions.

I now noticed you already have this explicit section below.  Perhaps we want to still rephrase this header to stress that the figure automation was all a new development as well.